usaspending_api/etl/management/commands/load_multiple_submissions.py

```diff
@@ -88,33 +87,40 @@ def handle(self, *args, **options):
 
                 if certify_date > most_recently_loaded_date:
                     missing_submissions.append((submission_id, agency_name, certify_date, most_recently_loaded_date))
-            except Exception as error:  # NOQA
-                logger.debug("Submission {} failed in pull from broker: {}".format(submission_id, error))
-                failed_submissions.append(submission_id)
-        logger.info("Total missing submissions: {}".format(len(missing_submissions)))
-        logger.info("-----------------------------------")
-        for next_missing_sub in missing_submissions:
-            logger.info(
-                "Submission ID {} ({})\tCertified: {}".format(
-                    next_missing_sub[0], next_missing_sub[1], next_missing_sub[2].date()
+            except Exception:
+                logger.exception("Submission ID {} failed in pull from broker".format(submission_id))
```

Comment:
This will make a huge list of exceptions to scroll through. It might make more sense to put the exception on line 99 and leave this as debug.

Reply:
It is extremely helpful to have the stacktrace then first time a submission fails. These are operation scripts, not API view logic so the logging needs are a little different.

Usually there isn't much to log, every night when it runs there are typically 0 DABS to load. Only during submission windows do the scripts run the load operations. When they fail, it's nice to see why right away instead of running the script again in a testing branch

Reply:
```suggestion
                logger.info("Submission ID {} failed in pull from broker".format(submission_id))
```
I was recommending switching which one was the error and which one was info, not removing this log entirely. It is a subjective matter of convenience, of course.

Reply:
logger.exception provides the stacktrace, logger.info will only report the provided text. 

Perhaps you can clarify your idea to report the "exception on lines 95-99"

Reply:
If exception automatically gives you the stacktrace, I now concede my idea altogether. Once again my Java shows